cmd/flow_capture.go

```diff
@@ -25,13 +25,7 @@ var flowCmd = &cobra.Command{
 }
 
 func runFlowCapture(_ *cobra.Command, _ []string) {
-	go func() {
-		if !scanner() {
-			return
-		}
-		// scanner returns on exit request
-		os.Exit(0)
-	}()
+	go scanner()
```

Comment:
`scanner` return code seems always ignored, if that's on purpose I'd suggest to make it not returning anything then

Reply:
sure, https://github.com/netobserv/network-observability-cli/pull/293/commits/704309456211cabb926f246015d4a1a1674ce68c